src/ducks/albums/components/AlbumToolbar.jsx

```diff
@@ -6,19 +6,24 @@ import { withRouter } from 'react-router'
 import { translate } from '../../../lib/I18n'
 
 import DeleteButton from '../../../components/DeleteButton'
+import ShareButton from '../../../components/ShareButton'
 import Alerter from '../../../components/Alerter'
 import Menu, { MenuButton, Item } from 'react-bosonic/lib/Menu'
 
 import { showSelectionBar } from '../../../actions'
 import { mustShowSelectionBar } from '../../../reducers'
-import { deleteAlbum } from '..'
+import { deleteAlbum, getAlbumShareLink } from '..'
 import DestroyConfirm from '../../../components/DestroyConfirm'
 import confirm from '../../../lib/confirm'
+import ShareModal from '../../../containers/ShareModal'
 
 import classNames from 'classnames'
 
-export const AlbumToolbar = ({ t, album, disabled = false, uploadPhotos, deleteAlbum, selectItems, params }) => (
+export const AlbumToolbar = ({ t, album, disabled = false, uploadPhotos, deleteAlbum, selectItems, shareAlbum }) => (
   <div className={styles['pho-toolbar']} role='toolbar'>
+    <ShareButton
```

Comment:
It's not supposed to be in the AlbumToolbar.jsx but in the AlbumPhotoToolbar.jsx, is it? The Share button is only visible in the view of the content of an album, not in the view that list all the available albums.